### DISCUSSION TITLE
Column layout not working

### DISCUSSION THREAD

**Discussion by thadnoble** [jdan/98.css#120] [Q&A] [answered]
I'm fairly new with HTML and CSS and I cannot for the life of me figure out how to fix this issue I've been having. I want my site to have a two-column layout and it worked before I imported the 98.css library, but after importing it the layout of the site broke and windows would be stacked on top of each other. 
![image](https://user-images.githubusercontent.com/92632994/138332072-e3e3edee-a8c5-4d38-a7d9-9abfb8bb7db8.png)

I tried putting in the styles in the div itself as I thought that the external stylesheet that I made must be conflicting with the 98 stylesheet, but it didn't work.
![image](https://user-images.githubusercontent.com/92632994/138332154-16276d1a-26a9-46cf-9c89-05dff3d472ef.png)

Is there any fix to this?

**Comment by jdan** [comment]
Hey! Can you send me a repro of your code with something like
codepen.io?

On October 21, 2021, GitHub ***@***.***> wrote:
> I'm fairly new with HTML and CSS and I cannot for the life of me
> figure out how to fix this issue I've been having. I want my site to
> have a two-column layout and it worked before I imported the 98.css
> library, but after importing it the layout of the site broke and
> windows would be stacked on top of each other.
>  
>

> I tried putting in the styles in the div itself as I thought that the
> external stylesheet that I made must be conflicting with the 98
> stylesheet, but it didn't work.
>  
>

> Is there any fix to this?
>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/jdan/98.css/discussions/120>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AACGEJDQWTBY3T3QNR7RPMDUIBIQZANCNFSM5GOW2DSA>.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-
email&mt=8&pt=524675> or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-
email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by thadnoble** [reply]
Here you go: https://codepen.io/telepathiq/pen/RwZoLJN

**Comment by jdan** [comment, marked as answer]
You'll want to close the `column_left` div before starting the `column_right`

![image](https://user-images.githubusercontent.com/287268/138447976-6f5a4391-a400-4ba3-8332-b42a35afb3e4.png)

An extra `</div>` on line 20 will do the trick! Check out the "Prettier" extension for your favorite editor and have it format your code on save - it'll fix the indentation of your code which can sometimes hide errors like these.
